Tests/XCUITests/JumpBackInTests.swift

```diff
@@ -0,0 +1,182 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+class JumpBackInTests: BaseTestCase {
+    func closeKeyboard() {
+        waitForExistence(app.buttons["urlBar-cancel"], timeout: TIMEOUT)
+        navigator.performAction(Action.CloseURLBarOpen)
+        navigator.nowAt(NewTabScreen)
+    }
+
+    func scrollDown() {
+        if !isTablet {
+            while app.staticTexts["Switch Your Default Browser"].exists || app.buttons["Learn How"].exists {
+                app.collectionViews["FxCollectionView"].swipeUp()
+            }
+        }
+    }
+
+    override func setUp() {
+        super.setUp()
+
+        closeKeyboard()
+
+        // "Jump Back In" is enabled by default. See Settings -> Homepage
+        navigator.goto(HomeSettings)
+        XCTAssertEqual(app.switches["Jump Back In"].value as! String, "1")
+
+        navigator.goto(NewTabScreen)
+    }
+
+    func testJumpBackInSection() {
+        // Open a tab and visit a page
+        navigator.openURL("https://www.example.com")
+        waitUntilPageLoad()
+
+        // Open a new tab
+        navigator.goto(TabTray)
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        closeKeyboard()
+
+        // "Jump Back In" section is displayed
+        waitForExistence(app.staticTexts["Jump Back In"])
+
+        // The contextual hint box is not displayed consistently, so
+        // I don't test for its existence.
+    }
+
+    func testGroupedTabs() {
+        // Enable "Tab Groups" from Settings -> Tabs
+        navigator.goto(TabsSettings)
+        navigator.performAction(Action.ToggleTabGroups)
+        navigator.goto(SettingsScreen)
+        app.navigationBars.buttons["Done"].tap()
+        navigator.nowAt(NewTabScreen)
+
+        // Create 3 groups in tab tray
+        let groups = ["test1", "test2", "test3"]
+        for group in groups {
+            for _ in 1...3 {
+                navigator.goto(TabTray)
+                navigator.performAction(Action.OpenNewTabFromTabTray)
+                navigator.openURL(group)
+                waitUntilPageLoad()
+            }
+        }
+
+        // Open a new tab
+        navigator.goto(TabTray)
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        closeKeyboard()
+
+        // Tap on the "test3" from "Jump Back In" section
+        scrollDown()
+        waitForExistence(app.staticTexts["Jump Back In"])
```

Comment:
This test fails here on Bitrise... I think the issues is the same as commented below.. we are in the tab tray but for some reason the '+' button is not clicked to go to the Homepanel view and so the check for the Jump Back In text fails